featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -176,6 +176,7 @@ def test_saveprogress(entityset):
     cutoff_time = pd.DataFrame({'time': times, 'instance_id': range(17)})
     property_feature = IdentityFeature(entityset['log']['value']) > 10
     save_progress = tempfile.mkdtemp()
+    property_feature.default_value
```

Comment:
The default value of `property_feature` seems to no longer be tested, in spite of no associated changes to that file. I'm testing it directly here, but we should probably find out the root cause. (tagging @rwedge)